core/src/test/java/org/opensearch/sql/expression/function/RelevanceFunctionResolverTest.java

```diff
@@ -0,0 +1,59 @@
+package org.opensearch.sql.expression.function;
```

Comment:
Missing copyright.

Reply:
Addressed in https://github.com/Bit-Quill/opensearch-project-sql/pull/99/commits/277ee6ad9485d4a0ad3097cec7af916894e14284